public/index.html

```diff
@@ -30,7 +30,7 @@
             const botConnection = new BotChat.DirectLine({
                 secret: params['s'],
                 token: params['t']
-                }, params["domain"]
+                }, params["domain"] //the domain should be the DL url https://directline.botframework.com/v3/directline
```

Comment:
Also, `params` is _initialized_ to empty, but is then filled in based on the arguments in the querystring. The point of this file is to allow simple testing of WebChat by passing in arguments via the URL.